context/app/static/js/components/detailPage/ContributorsTable/utils.ts

```diff
@@ -84,28 +84,50 @@ export const normalizeContributor = (contributor: ContributorAPIResponse): Contr
     ? normalizeCEDARContributor(contributor)
     : normalizeLegacyContributor(contributor);
 
+/**
+ * Given a contributor, determine if they are a contact. Necessary to account
+ *  for different versions of contributors schemas.
+ * @author Austen Money
+ * @param contributor a contributor to be checked.
+ * @returns true if the contributor is a contact, false otherwise.
+ */
+export const contributorIsContact = (contributor: Contributor, contacts: Contributor[]): boolean => {
+  switch (true) {
+    case contributor.isContact:
+      return true;
+    case contacts.find((contact) => contact.orcid === contributor.orcid) !== undefined:
```

Comment:
If there is a reason for this can we be more specific in the jsdoc?

Reply:
There is a case where the contributor object is missing the `is_contact` field and there exists a `contacts` object. If there is a `contact` with an `orcid` that matches the `orcid` of a `contributor`, that `contributor` should be treated as if `is_contact` is `true`. This is the case this PR is meant to address.

Agree it should be documented via a comment.

Reply:
Yes, this is a documented case, agreed about jsdoc clarity. Update:
```/**
 * Given a contributor, determine if they are a contact. Necessary to account
 *  for different versions of contributors schemas - some versions include the 
 *  isContact field in contributors, and some have a separate contacts array that
 *  must be checked.
 * @author Austen Money
 * @param contributor a contributor to be checked.
 * @param contacts an array of contacts to search for the contributor.
 * @returns true if the contributor is a contact, false otherwise.
 */```